src/core_init_atmosphere/Registry.xml

```diff
@@ -583,6 +594,9 @@
                         <var name="q2" packages="met_stage_out"/>
                         <var name="rh2" packages="met_stage_out"/>
                         <var name="t2m" packages="met_stage_out"/>
+                        <var name="nifa_gocart_clim" packages="met_stage_out;mp_thompson_aers_in"/>
+                        <var name="nwfa_gocart_clim" packages="met_stage_out;mp_thompson_aers_in"/>
+                        <var name="pwif_gocart_clim" packages="met_stage_out;mp_thompson_aers_in"/>
```

Comment:
I think we might have discussed this, but could you remind me why we write `nifa_gocart_clim`, `nwfa_gocart_clim`, and `pwif_gocart_clim` to the initial conditions file?

Reply:
Yes, we did discuss this earlier last week. Keeping those three fields in the init file was an easy way to use them to build the lateral boundary conditions for nifa and nwfa. We discussed that it may not be the ideal way for DA, but that we could review that issue after the 8.2 release. One option would be to add them to the invariant stream instead.

With these proposed changes, I was able to create the LBC and run regional MPAS with and without the gocart aerosols.

Reply:
Ok, thanks for the (re-)explanation!